neuralmonkey/readers/plain_text_reader.py

```diff
@@ -46,8 +46,8 @@ def reader(files: List[str]) -> Iterable[List[str]]:
                 warn("There is a missing column number {} in the dataset."
                      .format(column))
                 yield []
-
-            yield parsed_csv[0][column - 1].split(' ')
+            else:
+                yield parsed_csv[0][column - 1].split(' ')
 
```

Comment:
protoze kdyz sloupec chybi, tak to prvne yieldne [] a pak se to pokusi yieldnout i z neexistujicicho sloupce